### PR TITLE
Configure VM host network via nmstate

### PR DESCRIPTION
**Pull request by nocturnalastro**

### CODE REVIEW

**Review by crwr45 — APPROVED** [review]

**Review by arjuhe — COMMENTED** [review]
I've been avoiding it, but I'll start this now.

File: roles/apply_nmstate/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+- name: Install nmstate
+  package:
+    name: nmstate
+    state: present
+
+- name: "Copy nmstate_yaml to {{ nmstate_config_path }}"
```
Comment (mkochanowski) [inline roles/apply_nmstate/tasks/main.yml:6]: Would `nmstate_rendered_yaml` be a better name for `nmstate_yaml`? As far as I understand, the variable holds a rendered template for _nmstate_.

File: roles/setup_vm_host_network/defaults/main.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+vm_bridge_ip: "{{ machine_network_cidr | ipaddr('next_usable') }}"
+vm_bridge_prefix: "{{ machine_network_cidr | ipaddr('prefix') }}"
+bridge_name: "{{ cluster_name }}-br"
```
Comment (mkochanowski) [inline roles/setup_vm_host_network/defaults/main.yml:3]: Missing `vm_` prefix here

File: roles/process_nmstate/defaults/main.yml [hunk 3]
```diff
@@ -0,0 +1 @@
+target_name: "{{ inventory_hostname }}"
```
Comment (mkochanowski) [inline roles/process_nmstate/defaults/main.yml:1]: With `{{ inventory_hostname }}` var assignments like these, I think it'd be a great practice to leave one or two sentences explaining why that's required for each scenario. This comment could make it a little easier for the reader to understand the execution flow.

**Review by mkochanowski — CHANGES_REQUESTED** [review]

**Review by mkochanowski — APPROVED** [review]

**Review by arjuhe — APPROVED** [review]

**Pull request merged**